src/dashboard.tsx

```diff
@@ -116,7 +116,25 @@ export class DaskDashboardLauncher extends Widget {
     this.addClass('dask-DaskDashboardLauncher');
     this._items = options.items || DaskDashboardLauncher.DEFAULT_ITEMS;
     this._launchItem = options.launchItem;
-    this._input.urlChanged.connect(this.update, this);
+    this._input.urlChanged.connect(this.updateLinks, this);
+  }
+
+  protected async updateLinks() {
```

Comment:
Do you envision subclasses making use of this function? If not, I think we should make it private: `private async _updateLinks(): Promise<void>`

Reply:
I'm not experienced enough with Jupyter Labextensions to predict if DaskDashboardLauncher will ever be subclassed. Probably not. :) I'll make that change.

Reply:
👍